apps/api/v1/test/lib/middleware/verifyApiKey.test.ts

```diff
@@ -20,20 +20,21 @@ afterEach(() => {
   vi.resetAllMocks();
 });
 
-vi.mock("@calcom/features/ee/common/server/checkLicense", () => {
-  return {
-    default: vi.fn(),
-  };
-});
-
-vi.mock("~/lib/utils/isAdmin", () => {
-  return {
-    isAdminGuard: vi.fn(),
-  };
-});
+vi.mock("@calcom/prisma");
+vi.mock("~/lib/utils/isAdmin", () => ({
+  isAdminGuard: vi.fn(),
+}));
 
 describe("Verify API key", () => {
-  it("It should throw an error if the api key is not valid", async () => {
+  let service: LicenseKeyService;
+
+  beforeEach(async () => {
+    service = await LicenseKeyService.create();
+
+    vi.spyOn(service, "checkLicense");
+  });
+
```

Comment:
I don't think this will work for parallel test runs doesn't it?

Reply:
I think it should? WIll look into it